### PR TITLE
[CARBONDATA-830] Incorrect schedule for NewCarbonDataLoadRDD

### PR DESCRIPTION
**Pull request by Sephiroth-Lin**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[CARBONDATA-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
 - [ ] Testing done
 
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - What manual testing you have done?
        - Any additional information to help reviewers in testing this change.
         
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
                 
---

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by Sephiroth-Lin** [comment]
@QiangCai Please help review this

File: integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/NewCarbonDataLoadRDD.scala [hunk 1]
```diff
@@ -292,12 +292,7 @@ class NewCarbonDataLoadRDD[K, V](
         }
       }
 
-      val sortedList = blockMap.entrySet().asScala.toSeq.sortWith { (nodeCount1, nodeCount2) =>
-        nodeCount1.getValue > nodeCount2.getValue
-      }
-
-      val sortedNodesList = sortedList.map(nodeCount => nodeCount.getKey).take(2)
-      firstOptionLocation ++ sortedNodesList
+      firstOptionLocation
```
Comment (watermen) [inline integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/NewCarbonDataLoadRDD.scala]: Add some notes to explain why we do this.

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**